pkgs/development/python-modules/wrapcco/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  numpy,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "wrapcco";
+  version = "0.1.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-nts5cujixB+PfAf9Fk7kOXW4BeI/0q5GYTthCI76ZOA=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [
+    numpy
+    setuptools
+  ];
+
+  doCheck = false;
```

Comment:
Please add a comment why tests are disabled, this is a common practice we do.

Reply:
Also, add a pythonImportsCheck to check if the module can be imported correctly

Reply:
Actually, check the patch that Nick has posted in this PR, it does everything correctly.